Dockerfile

```
@@ -35,6 +35,10 @@ LABEL org.opencontainers.image.source=https://github.com/approvers/OreOreBot2
 ENV NODE_ENV=production
 WORKDIR /app
 
+# Sentry に必要なパッケージ
+RUN apt-get update \
+    && apt-get install -y libssl-dev ca-certificates
```

```suggestion
    && apt-get install -y --no-install-recommends libssl-dev=3.0.2-0 ca-certificates=20230311 \
    && apt-get clean \
    && rm -rf /var/lib/apt/lists/*
```
hadolint でも警告が出ていますが, バージョン固定, インストール推奨の拒否, インストール後の掃除をしておきましょう

---

すいません, バージョン指定違っていたようなので検索して最新の使えるバージョンを入れてほしいです